artemis-cli/src/main/java/org/apache/activemq/artemis/cli/commands/queue/UpdateQueue.java

```diff
@@ -31,7 +31,7 @@
    String name;
 
    @Option(name = "--deleteOnNoConsumers", description = "whether to delete when it's last consumers disconnects)")
-   Boolean deleteOnNoConsumers = null;
+   String deleteOnNoConsumers = null;
```

Comment:
I had a similar case somewhere else.  Where I created two different Booleans. And it would be an error to have both. 


--optionA or --noOptionA



The input could then be used if none is used.